docs/_static/style.css

```diff
@@ -15,6 +15,10 @@ body {
   padding: 0;
 }
 
+body.sans {
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
```

Comment:
I'd like to point out the existence of [`system-ui`](https://caniuse.com/#feat=font-family-system-ui) here, although [it can be a little problematic](https://infinnie.github.io/blog/2017/systemui.html). This font stack is good, though.